programs/test/benchmark.c

```diff
@@ -150,6 +150,16 @@ do {                                                                    \
 
 #if defined(MBEDTLS_MEMORY_BUFFER_ALLOC_C) && defined(MBEDTLS_MEMORY_DEBUG)
 
+/* How much space to reserve for the title when printing heap usage results.
+ * Updated manually as the output of the following command:
```

Comment:
What about to add a comment explaining what this command is computing? From what I understand this is the length of the longest title + 2.